whitepaper/chapters/transactions.tex

```diff
@@ -16,6 +16,8 @@ \section{Transactions}
 Aggregate transactions allow basic transactions to be combined into potentially complex operations and executed atomically.
 This increases developer flexibility relative to a system that only guarantees atomicity for individual operations while still constraining the global set of operations allowed to a finite set.
 It does not require the introduction of a Turing-complete language and all of its inherent disadvantages.
+Developers do not need to learn any new languages nor develop bespoke contract implementations from scratch.
+Simple composition should be much less error prone and lead to fewer bugs.
```

Comment:
I'd avoid using simple and highlight the comparison. For example, you could write:

``The composition of transactions should be less error-prone and lead to fewer bugs than implementing computationally complete programs.``